docs/en/connector-v2/source/RocketMQ.md

```diff
@@ -218,3 +219,62 @@ sink {
 }
 ```
 
+### Specified timestamp Simple:
+
+> This is to specify a time to consume, and I dynamically sense the existence of a new partition every 1000 milliseconds to pull the consumption
```

Comment:
Please update the description

Reply:
Thank you for your reminder, I neglected it.